data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/validator/PostAttestation.java

```diff
@@ -54,12 +53,13 @@ public PostAttestation(final ValidatorDataProvider provider, final JsonProvider
   @OpenApi(
       path = ROUTE,
       method = HttpMethod.POST,
-      summary = "Submit a signed attestation.",
+      summary = "Submit a signed attestation",
       tags = {TAG_VALIDATOR},
       requestBody = @OpenApiRequestBody(content = {@OpenApiContent(from = Attestation.class)}),
       description =
           "Submit a signed attestation to the beacon node to be validated and submitted if valid.\n\n"
-              + "This endpoint does not protected against slashing. Signing the attestation can result in a slashable offence.",
+              + "This endpoint does not protected against slashing. Signing the attestation can result in a slashable offence.\n\n"
+              + "Deprecated - use `/eth/v1/validator/attestation_data` instead.",
       responses = {
```

Comment:
Don't we normally add a `deprecated = true` option?

Reply:
yep. done